opengrok-indexer/src/main/java/org/opengrok/indexer/history/BazaarHistoryParser.java

```diff
@@ -179,7 +179,7 @@ public void processStream(InputStream input) throws IOException {
                     }
                     break;
                 default:
-                    LOGGER.log(Level.WARNING, "Unknown parser state: {0}", state);
+                    LOGGER.log(Level.FINEST, "Unknown parser state: {0}", state);
```

Comment:
"unknown" is not a keyword?

Reply:
No, it's not. I can add.

Reply:
The question is not whether you *can* add but *should* you.